lib/krane/errors.rb

```diff
@@ -1,8 +1,10 @@
 # frozen_string_literal: true
+require 'kubernetes-deploy/errors'
+
 module Krane
   class FatalDeploymentError < StandardError; end
   class FatalKubeAPIError < FatalDeploymentError; end
-  class KubectlError < StandardError; end
+  class KubectlError < FatalDeploymentError; end
```

Comment:
Did you mean to change this? Kubectl errors aren't necessarily fatal (though if it raised out all the way to the user, obviously it was!)

Reply:
No, it was not intended - thanks for catching this one.